crates/libcontainer/src/container/container_delete.rs

```diff
@@ -27,9 +27,8 @@ impl Container {
     pub fn delete(&mut self, force: bool) -> Result<()> {
         self.refresh_status()
             .context("failed to refresh container status")?;
-        if self.can_kill() && force {
-            self.do_kill(signal::Signal::SIGKILL, true)?;
-            self.set_status(ContainerStatus::Stopped).save()?;
```

Comment:
Any reason this `set_status` is removed? Do we still want to set the status to Stopped?

Reply:
It is not recommended to delete this logic, We should follow the runtime-spec.

Attempting to delete a container that is not [stopped](https://github.com/opencontainers/runtime-spec/blob/main/runtime.md#state) MUST have no effect on the container and MUST [generate an error](https://github.com/opencontainers/runtime-spec/blob/main/runtime.md#errors).

https://github.com/opencontainers/runtime-spec/blob/main/runtime.md#delete

Reply:
@lengrongfu Thank you for pointing this out. I believe you are right. See my PR: https://github.com/containers/youki/pull/1654 We may also wish to submit a PR to the OCI spec to make the change since `runc` has implemented this behavior as early as 2017.